tests/AutoReview/ProjectCodeTest.php

```diff
@@ -65,6 +65,7 @@
         \PhpCsFixer\Runner\FileFilterIterator::class,
         \PhpCsFixer\Runner\FileLintingIterator::class,
         \PhpCsFixer\StdinFileInfo::class,
+        \PhpCsFixer\Test\Assert\AssertTokensTrait::class,
```

Comment:
I'm not a big fan of testing "internal" things, like checking private properties or calling private methods. IMO we should only test public things. If a trait has a public method, then we can test it on the class that use it directly.